### DISCUSSION TITLE
globaltoc/globalnavigation/increase maxdepth option for navbar

### DISCUSSION THREAD

**Discussion by casperdcl** [pradyunsg/furo#242] [Ideas] [answered]
Rather than just rendering (sub)pages in the nav bar, ideally there should be another template (e.g. `globalnavigation.html`) to show `<h2>` etc links.

Similar to the builtin theme's `localtoc.html` vs `globaltoc.html` concept.

**Comment by pradyunsg** [comment, marked as answer]
From https://github.com/pradyunsg/furo/issues/79:

> I'm not going to be adding this since separating the "page structure" from "site structure" in Furo, is an explicit design choice. The left sidebar is for presenting site structure and the right sidebar is for presenting the page content hierarchy. I find the mixing of the two in other Sphinx themes to be a suboptimal experience when looking for information.

**Comment by kislyuk** [comment]
> I'm not going to be adding this since separating the "page structure" from "site structure" in Furo, is an explicit design choice.

I would urge a more flexible approach here. Many doc sites are single-page sites where the local TOC *is* the site structure. I love furo's capabilities, but I don't think having a blank or meaningless left sidebar and a smaller, less noticeable right sidebar with the actual TOC works in this case.

I have been able to work around this design limitation by declaring an `html_sidebars` as follows:
```
templates_path = ["_templates"]
html_sidebars = {
    "**": [
        "sidebar/brand.html",
        "sidebar/search.html",
        "sidebar/scroll-start.html",
        "_templates/toc.html",
        "sidebar/scroll-end.html",
    ]
}
```
where `_templates/toc.html` contains just `{{toc}}`. But I would love to see a more official way to configure single-page doc sites. Please let me know if I'm missing something.
